src/abstractdataframe/abstractdataframe.jl

```diff
@@ -438,8 +438,7 @@ completecases(df)
 ```
 
 """
-function completecases(df::AbstractDataFrame)
-    ## Returns a Vector{Bool} of indexes of complete cases (rows with no NA's).
+function completecases(df::AbstractDataFrame)::AbstractVector{Bool}
```

Comment:
Notice that there is only a single assignment because of the `.&=`. However, it might make sense just to initialize `res` to be the type we'd like and the extend the loop range to include `1`.